### PR TITLE
[Defensive] Add Command

### PR DESCRIPTION
**Pull request by timjkong**

### CODE REVIEW

File: src/test/java/seedu/address/logic/command/add/AddAccommodationCommandTest.java [hunk 1]
```diff
@@ -112,6 +113,13 @@ public void addTravelPlanObject(TravelPlanObject travelPlanObject) {
             accommodationsAdded.add(accommodation);
         }
 
+        @Override
```
Comment (jeannetoh99) [inline src/test/java/seedu/address/logic/command/add/AddAccommodationCommandTest.java:116]: Not this line, but for line 97, can change "person" in the javadoc comment to accommodation?

**Review by jeannetoh99 — APPROVED** [review]
LGTM, just one comment.

**Review by lyeyixian — APPROVED** [review]
LGTM

**Review by jeannetoh99 — APPROVED** [review]
LGTM

**Pull request merged**